Dockerfile

```diff
@@ -7,6 +7,7 @@ COPY ./ .
 RUN npm run build
 
 FROM nginx as production-stage
-RUN mkdir /app
+EXPOSE 9090
```

Comment:
In fact IANA now recognizes 8080 as an alternate HTTP port: https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=8080

Reply:
I agree, but was following the standard port used throughout the code and didn't want it to be inconsistent. Also not sure if it would break things. If Simon and Eric are okay with changing to 8080 I can make that change.

Reply:
I used 8080 in the docker image publish PR:

- #312

Reply:
All good with 8080!